src/CONST.js

```diff
@@ -576,6 +576,10 @@ const CONST = {
             DAILY: 'daily',
             ALWAYS: 'always',
         },
+        WRITE_CAPABILITIES: {
```

Comment:
Do you know if we plan to use this setting for non-room-related things? If not yet, maybe a comment could be nice to explain what these relate to